docs/api/app.md

```diff
@@ -1468,6 +1468,111 @@ details.
 
 **Note:** Enable `Secure Keyboard Entry` only when it is needed and disable it when it is no longer needed.
 
+### `app.setProxy(config)`
```

Comment:
Would we want to name this method more descriptively to distinguish it from `ses.setProxy`? Something like `app.setSessionlessProxy` or `app.setProxyWithoutSession`?

Reply:
I prefer the consistency in the API names. The scope of the methods clearly define the boundaries between the two methods, also docs are clear on the difference.